src/mappings/modules/liquidation/dog.ts

```diff
@@ -66,3 +67,35 @@ export function handleFileClip(event: File3): void {
     ilk.save()
   }
 }
+
+export function handleBark(event: Bark): void {
+  let idStr = event.params.id.toString()
+  let vaultId = event.params.urn.toHexString() + '-' + event.params.ilk.toString()
+  let collateralTypeId = event.params.ilk.toString()
+  let saleAuction = saleAuctions.loadOrCreateSaleAuction(idStr, event)
+  saleAuction.vault = vaultId
+  saleAuction.collateralType = collateralTypeId
+  saleAuction.clipperContract = event.params.clip
+  saleAuction.save()
+
+  let dogContract = Dog.bind(Address.fromString('0x135954d155898D42C90D2a57824C690e0c7BEf1B'))
+  let callResult = dogContract.try_Dirt()
+  let callResult2 = dogContract.try_ilks(event.params.ilk)
+  let ilkDirt = units.fromRad(callResult2.value.value3)
+
+  if (callResult.reverted) {
+    log.warning('handleBark: try_Dirt reverted', [])
+  }
+  let amount = units.fromRad(callResult.value)
+  let system = systemModule.getSystemState(event)
+  system.totalDaiAmountToCoverDebtAndFees = amount
```

Comment:
```
uint256 tab = mul(due, milk.chop) / WAD;
Dirt = add(Dirt, tab);
```

I guess we can calculate the `tab` because we have the `due` and the `chop`. Then we can add `tab` to the current `Dirt` and update the `totalDaiAmountToCoverDebtAndFees`